### PR TITLE
[#6060] academy_landing_page: style and structure landing page intro

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

File: apps/academy/templates/academy/academy_landing_page.html [hunk 1]
```diff
@@ -5,15 +5,17 @@
 
 {% endblock %}
 
-{% block menu-color %}style="color: {{ page.textcolor }};"{% endblock %}
-
 {% block content %}
-<section class="intro-screen" style="color: {{ page.textcolor }}; background-image: linear-gradient(148deg, {{ page.color1 }}, {{ page.color2 }});">
+<section class="simple-header-bg" style="color: {{ page.textcolor }}; background-image: linear-gradient(148deg, {{ page.color1 }}, {{ page.color2 }});">
```
Comment (khamui) [inline apps/academy/templates/academy/academy_landing_page.html:9]: the bottom padding looks a little bit too big for desktop and mobile.

**Review by khamui — COMMENTED** [review]
Nice! just one thing:

File: apps/academy/templates/academy/academy_landing_page.html (hunk 1 above)
Comment (phillimorland) [reply apps/academy/templates/academy/academy_landing_page.html:9]: i didn't do it pixel perfect to designs as they are not using the same spacing ratios but it's alot closer! :p

**Review by fuzzylogic2000 — APPROVED** [review]
NIce!

**Pull request merged**